sessionctx/variable/statusvar.go

```diff
@@ -15,377 +15,85 @@ package variable
 
 import (
 	"strings"
+	"sync"
+
+	"github.com/juju/errors"
 )
 
-// StatusVars is global status vars map.
-var StatusVars map[string]*SysVar
+// statusVars is global status vars map.
+var statusVars map[string]*StatusVal
+var globalStatusScopes = make(map[string]ScopeFlag)
+var defaultScopeFlag = ScopeGlobal | ScopeSession
+var statistMu sync.RWMutex
 
-// GetStatusVar returns status var infomation for name.
-func GetStatusVar(name string) *SysVar {
-	name = strings.ToLower(name)
-	return StatusVars[name]
+// Statists is the set of all statists.
+var Statists = make(map[string]Statist)
+
+// StatusVal is the value of the corresponding status variable.
+type StatusVal struct {
+	Scope ScopeFlag
+	Value interface{}
+}
+
+// Statist is the interface of statist.
```

Comment:
It means statistician. I think is shorter than statistician, but it is less used.
I will replace it with statistics.